src/ServerRequest.php

```diff
@@ -180,23 +180,35 @@ public static function fromGlobals()
             ->withUploadedFiles(self::normalizeFiles($_FILES));
     }
 
+    private static function extractPortFromHost($host)
```

Comment:
please name this `extractHostAndPortFromAuthority` which would use the correct terms according to uri spec.